### ISSUE TITLE
Merchants are still experiencing Shipping Zone errors with release of 11.1 

### ISSUE THREAD

**Issue by AmyAtMollie** [mollie/OpenCart#295]
Merchants have still been experiencing issues with Shipping Zone when trying to make payments. Merchants have upgraded to 11.1 and are still having the same issue. This bug was fixed according to the latest release. 
One merchant gets the following error after updating to 11.1: 
"An error occurred while setting up the payment at Mollie. Click the button below to return to the checkout screen.
I don't get any errors in my error log. 
I only see the following message at Mollie > Support when I try to pay:
Mollie Payment Error: Mollie payment require payment and shipping address details. Empty required field: Shipping Zone
"
Second error: "Merchant receives error: Mollie payment require payment and shipping address details. Empty required field: Shipping Zone. Saw this was a bug in 11.1.0 but the merchant already updated to 11.1.1 but this still gives the error."

I am unable to reproduce the error. Is there something that our merchants need to do to fix the issue?

**Comment by AmyAtMollie** [comment]
Hello are there any updates on this issue?

**Comment by QualityWorks** [comment]
we have tested this and the issue seems fixed. We can help the merchant to update to the latest version?

**Comment by AmyAtMollie** [comment]
Hello QualityWorks, 
The merchant was still reproducing the error after updating to 11.1. We tested the issue from both of our sides and it appears the issue is occurring with registered accounts that were created a long time ago. What we think is happening is that the time these accounts were created, the 'regio/provincie' field wasnt recquired yet and so it was never stored in their address details. The checkout still does a check of this field and throws an error for these older accounts.
When I tested with an unregistered account I did not get the shipping zone error. When I made a new account I did not get the error either. For the merchant with an old account, they are getting this error. 
Is this issue known and will there be a fix for it?
